storage.go

```diff
@@ -59,9 +59,6 @@ func (s *Storage) list(ctx context.Context, path string, opt pairStorageList) (o
 					o.Mode |= ModeRead
 				case ipfs.TDirectory:
 					o.Mode |= ModeDir
-				case ipfs.TSymlink:
```

Comment:
From my testing it seems that currently IPFS MFS does not support `symink` type and the related issue is open, I think I should remove this condition because there is no way to add `symlink` type files.

Maybe I'm wrong, but I didn't find a more definite conclusion.

Reply:
> I think I should remove this condition because there is no way to add symlink type files.

It's OK. We don't need to implement features that don't have native support.